radixdlt/src/integration/java/com/radixdlt/consensus/simulation/network/SimulationNetwork.java

```diff
@@ -149,8 +155,8 @@ public static Builder builder() {
 	public BFTEventSender getNetworkSender(ECPublicKey forNode) {
 		return new BFTEventSender() {
 			@Override
-			public void broadcastProposal(Proposal proposal) {
-				for (ECPublicKey reader : receivers.keySet()) {
+			public void broadcastProposal(Proposal proposal, Set<ECPublicKey> nodes) {
+				for (ECPublicKey reader : nodes) {
```

Comment:
Is `nodes` supposed to be a subset of `receivers.keySet()` here? Does it even matter?

Reply:
I guess this is for the implementation of epochs, where validators for a given epoch will be a proper subset of all the network nodes.

Reply:
yup, subset